Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -53,6 +53,11 @@ protected CacheBase()
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether context should be maintained until type has been instantiated or not.
```

Comment:
This will probably need a much better summary to make it very clear to the devs using this as to what this property is actually being used for. Also, you need to update the docs to document this new property.